packages/xrpl/src/utils/hooks.ts

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
What are the uses of these utils?

Reply:
The purpose of these utils can be found here.

https://github.com/Transia-RnD/xls-playground/blob/main/ts/test/hooks.test.ts

I also have them in [this](https://github.com/Transia-RnD/xrpl-helpers/blob/main/ts/src/hooks/hooks.ts) repo here if we  dont want them in the sdk.